### PR TITLE
Swift SDK Event Library page: Incorporate tech review feedback

### PR DESCRIPTION
**Pull request by dacharyc**
## Pull Request Info

This PR incorporates tech review feedback left on #1917 

### Staged Changes (Requires MongoDB Corp SSO)

- [Event Library](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/event-library-tech-review-feedback/sdk/swift/event-library/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/swift/event-library.txt [hunk 1]
```diff
@@ -124,8 +124,8 @@ activities that occur within the scope of the event as read or write events.
   properties that change durign the scope of the event.
 
 Using ``beginScope`` to record an event opens the event {+realm+} if it 
-is not already open. This can throw the same errors that opening a {+realm+}
-can normally throw.
+is not already open. The SDK opens event {+realm+}s on the background
```
Comment (mongomoe) [inline source/sdk/swift/event-library.txt:127]: This sentence: `The SDK opens event realms on the background thread reports errors to the error callback.` just does not make sense to me, can you clarify it?

**Review by mongomoe — COMMENTED** [review]
1 quick q

File: source/sdk/swift/event-library.txt (hunk 1 above)
Comment (dacharyc) [reply source/sdk/swift/event-library.txt:127]: Oops! It's missing an `and` there - just updated it. The sentence should be:

> The SDK opens event {+realm+}s on the background thread and reports errors to the error callback.

**Review by mongomoe — APPROVED** [review]
LGTM

**Pull request merged**